src/IronWorkerRuntime.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace IronWorker
+
+use IronCore\IronCore;
+
+/**
+ * PHP runtime library for IronWorker
+ * IronWorker is a massively scalable background processing system.
+ * Use this class to access common functionality from inside an IronWorker task
+ *
+ * @link https://github.com/iron-io/iron_worker_php
+ * @link http://www.iron.io/
+ * @link http://dev.iron.io/
+ * @version 2.0.2
+ * @package IronWorker
+ * @copyright Feel free to copy, steal, take credit for, or whatever you feel like doing with this code. ;)
+ */
+class Runtime extends IronCore {
+
+  /**
+   * getConfig gets the configuration from the input to this worker
+   */
+  public static function getConfig($assoc = true)
```

Comment:
@arschles that's for compatibility with older code.
personally prefer having assoc=true all the time